src/components/Modal/index.tsx

```diff
@@ -191,9 +195,15 @@ export default function Modal({
                 style={props}
                 onDismiss={onDismiss}
                 initialFocusRef={initialFocusRef}
-                mobile={false}
+                mobile={undefined}
               >
-                <StyledDialogContent hidden={true} minHeight={minHeight} maxHeight={maxHeight} isOpen={isOpen}>
+                <StyledDialogContent
+                  hidden={true}
+                  minHeight={minHeight}
+                  maxHeight={maxHeight}
+                  isOpen={isOpen}
+                  aria-label=""
```

Comment:
is this label required with attrs above? can we do just one?

Reply:
Nope can remove this